### PR TITLE
clean up some REPL commands

### PR DESCRIPTION
**Pull request by KristofferC**
Fixes https://github.com/JuliaLang/Pkg.jl/issues/200

### CODE REVIEW

File: src/REPLMode.jl [hunk 1]
```diff
@@ -28,28 +28,18 @@ const cmds = Dict(
...
     "add"       => CMD_ADD,
-    "install"   => CMD_ADD,
-    "+"         => CMD_ADD,
     "rm"        => CMD_RM,
     "remove"    => CMD_RM,
-    "uninstall" => CMD_RM,
-    "-"         => CMD_RM,
     "up"        => CMD_UP,
     "update"    => CMD_UP,
-    "upgrade"   => CMD_UP,
```
Comment (StefanKarpinski) [inline src/REPLMode.jl]: Not sure if we should get rid of this one: the operation it's doing is an upgrade after all. I had proposed using the distinction between `update` and `upgrade` to differentiate getting a minor update versus doing a potentially-breaking `upgrade` but people didn't like the subtlety of the difference, which is why I made them aliases of each other, abbreviated to `up` for either one.

Comment (pfitzseb) [reply src/REPLMode.jl]: Yep, the only potentially annyoing thing about keeping both is that right now `up\tab` shows both. We could canonicalize that to `update` though.

Comment (KristofferC) [reply src/REPLMode.jl]: Sure it does an `upgrade` and `remove` does an `uninstall` but keeping a bunch of different verbs that are almost synonyms seems unnecessary. One short form and one long form should be enough.

Comment (StefanKarpinski) [reply src/REPLMode.jl]: Fair enough.

Comment (staticfloat) [reply src/REPLMode.jl]: Agreed; whenever I see both `update` and `upgrade` I get flashbacks to `brew` where they actually mean different things.

**Pull request merged**